docs/ADRs/0006-agile-tooling.md

```diff
@@ -0,0 +1,27 @@
+---
+    title: 0006 Agile tooling
+    adr:
+        author: Berry den Hartog
+        created: 28-Feb-2024
+        status:  accepted
+---
+
+## Context
+
+Our development team wants to enhance transparency and productivity in our software development processes.
+We are using GitHub for version control and collaboration. However, to further streamline our process, there is a need
+to incorporate tooling for managing the effort of our team.
+
+## Decision
+
+We will use GitHub Projects as our agile process tool
```

Comment:
```suggestion
We will use GitHub Projects as our agile process tool, in combination with Mattermost Boards to fill up the gaps Github Projects leaves behind for our Agile process.
```

Reply:
I would prefer to not add mattermost since it currenlty doest not add any agile tooling

Reply:
hmm sure, has we also removed the line below that we don't have any current limitations that we need to overcome, I am fine with it. And the role is very minor indeed, otherwise we make it larger than it actually is.